### PR TITLE
dnsdist: Add the 'clearConsoleHistory' command

### PR DESCRIPTION
**Pull request by rgacogne**
### Short description

It removes the history items and delta() data from memory, which might be useful when a lot of commands are issued over time, to prevent a large memory usage.

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: pdns/dnsdist-console.cc [hunk 1]
```diff
@@ -362,10 +360,11 @@ const std::vector<ConsoleKeyword> g_consoleKeywords{
   { "AndRule", true, "list of DNS rules", "matches if all sub-rules matches" },
   { "benchRule", true, "DNS Rule [, iterations [, suffix]]", "bench the specified DNS rule" },
   { "carbonServer", true, "serverIP, [ourname], [interval]", "report statistics to serverIP using our hostname, or 'ourname' if provided, every 'interval' seconds" },
-  { "controlSocket", true, "addr", "open a control socket on this address / connect to this address in client mode" },
```
Comment (phonedph1) [inline pdns/dnsdist-console.cc:365]: was this accidentally removed?

Comment (rgacogne) [reply pdns/dnsdist-console.cc:365]: It has been moved a few lines below because it was not in the right place before :-)

Comment (phonedph1) [reply pdns/dnsdist-console.cc:365]: Doh. I obviously didn't look close enough :D

**Comment by rgacogne** [comment]
This PR is lacking a documentation update!

**Pull request merged**